src/WebJobs.Script.Grpc/Capabilities/Capabilities.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Google.Protobuf.Collections;
+
+namespace Microsoft.Azure.WebJobs.Script.Grpc.Capabilities
```

Comment:
- Capabilities are per languager worker channel. Move this to Webjobs.script project under 'azure-functions-host\src\WebJobs.Script\Rpc\Capabilities.cs'
- Add Capabilites instance as a property of LanguageWorkerChannel

```cs
private Capabilities _capabilites = new Capabilities();
```